content/post/2019-04-29-cranlogs-2-1-1.Rmarkdown

```diff
@@ -0,0 +1,143 @@
+---
+title: cranlogs 2.1.1 is on CRAN!
+date: '2019-04-15'
+slug: cranlogs-2-1-1
+tags:
+  - cranlogs
+  - R package
+  - release
+---
+
+Version 2.1.1 of the `cranlogs` package has been [released on CRAN](https://cran.r-project.org/package=cranlogs)! `cranlogs` queries a [web API maintained by R-hub](https://github.com/r-hub/cranlogs.app#the-api-of-the-cran-downloads-database) that contains the daily download
+numbers for each package and for R itself from the RStudio CRAN mirror. :chart_with_upwards_trend:
+
+Get `cranlogs`'s latest version from CRAN, hopefully from RStudio CRAN mirror so we can monitor counts. :wink:
+
+```r
+install.packages("cranlogs", repos = "https://cran.rstudio.com/")
+```
+
+The [changes brought by this version](https://r-hub.github.io/cranlogs/news/index.html#cranlogs-2-1-1) mostly are a few docs and bug fixes :wave: :bug:, but also the addition of a brand-new function simplifying your use of cranlogs badges, [`cranlogs_badge()`](https://r-hub.github.io/cranlogs/reference/cranlogs_badge.html)! Since it is the first time we blog about `cranlogs` after its transfer to R-hub, we shall use the rest this blog post as an introduction to it.
+
+### `cranlogs`, how and why?
+
+### `cranlogs` data source and its quality
+
+[RStudio](http://www.rstudio.com) publishes the download logs from their
+CRAN package mirror daily at <http://cran-logs.rstudio.com>. The `cranlogs` package queries a [web API maintained by R-hub](https://github.com/r-hub/cranlogs.app#the-api-of-the-cran-downloads-database) that contains the daily download numbers for each package.
+
+The RStudio CRAN mirror is not the only CRAN mirror, but it’s a popular one: it’s the default choice for RStudio users. The actual number of downloads over all CRAN mirrors is unknown. Therefore, using numbers of downloads from that mirror is a _proxy_ for actual numbers of downloads, and when analysing it one has to assume the observed trends are representative of what happens via other mirrors.
+
+The data `cranlogs` queries has other limitations:
+
+* zero counts on days the RStudio server serving the log was down. 
+
+* count inflation due to CRAN-wide shared daily downloads
+
+* re-download/CRAN mirror spikes due to package updates
+
+The latter two downsides can be corrected by the use of the [`adjustedcranlogs` package ](https://github.com/tylermorganwall/adjustedcranlogs) by Tyler Morgan-Wall. Refer to that package documentation and code to find out more about it adjusts the download counts.
```

Comment:
TBH I am not sure that it makes sense to get rid of the downloads after updates. Eg. if a package has a blog post after release, then many people will update/install it, why would we want to ignore these numbers. 

Yes, there will be spikes in the download numbers, but that's just the nature of the data, we don't need to get rid of them. 

So I would write this differently, e.g. "If you are not interested in downloads after updates, then takes a look at the adjustedcranlogs package, which uses a model to filter them out" or sg like this.